github/ci/prow/files/config.yaml

```diff
@@ -180,6 +180,14 @@ branch-protection:
               protect: true
         kubectl-virt-plugin:
           protect: true
+    nmstate:
+      repos:
+        kubernetes-nmstate:
+          protect: true
+          required_status_checks:
+            contexts:
+              - pull-kubernetes-nmstate-unit-test
```

Comment:
these are not necessary to mention, branchprotection will add them automatically from the presubmits and make them required, if they are in the job definition.

Reply:
so if they are marked at 'required' at job definitions they will bee added right ?

Reply:
Yes, that should work. Please see that these are picked up for kubevirt/kubevirt also automatically. We just added travis and coverage.